charms/worker/charmcraft.yaml

```diff
@@ -171,6 +171,8 @@ requires:
     limit: 1
   azure:
     interface: azure-integration
+  certificates:
+    interface: tls-certificates
```

Comment:
can you update the terraform providers (for both charms) with these new relation endpoints?

Reply:
if necessary, it can be in another PR